### PR TITLE
Fix (=really add) Clinker

### PR DESCRIPTION
**Pull request by andrewlonsdale**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

These are minor changes intended to trigger the build process again as previous addition of this recipe passed tests and was merged into master (https://github.com/bioconda/bioconda-recipes/pull/10244)  but the recipe does not appear to be available.

### CODE REVIEW

**Comment by epruesse** [comment]
The build log shows "nothing to be done". This still won't build anything.

**Comment by andrewlonsdale** [comment]
Thanks for looking at this @epruesse 

I'm still unclear what is going on with this recipe. Can you advise on what is preventing this recipe from appearing in bioconda, i.e not available from conda or here https://bioconda.github.io/recipes.html, but appears in the master branch.

**Comment by tfursten** [comment]
@andrewlonsdale I had this 'nothing to be done' issue, I think it was caused by naming the file meta.yml instead of meta.yaml.

**Comment by epruesse** [comment]
Aaaah! Thanks @tfursten - that seems to be the case here too.

**Comment by andrewlonsdale** [comment]
Thanks @tfursten and @epruesse! That 'meta.yml' fix was helpful, revealing underlying issues with recipe. I've had some timeout issues with local circleci as we as the auto github tests, latest update hopefully addresses these.

**Comment by epruesse** [comment]
You probably have something in the recipe that gets stuck. Your job timed out after 5 hours of processing.

File: recipes/clinker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  number: 1
+  noarch: generic
+
+requirements:
+  run:
+    - bpipe
+    - samtools
+    - star ==2.5.3a
+    - bioconductor-biomart ==2.34.2
+    - bioconductor-gviz ==1.22.3
```
Comment (epruesse) [inline recipes/clinker/meta.yaml]: If you need to pin that tightly, you should distribute an environment file. Please make these something like 

```
 - star >=2.5.3a, <3
 - bioconductor-biomart >=2.34.2, <3
 - bioconductor-gviz >=1.22.3, <2
```

File: recipes/clinker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Oshlack/Clinker/archive/v{{ version }}.tar.gz
+  sha256: "5dbb97da27f29f53a8282e456ba0093bbdde292eb9e76405434117da4d1a76be"
+
+build:
+  number: 1
```
Comment (epruesse) [inline recipes/clinker/meta.yaml]: Should be 0  - no packages exist yet.

File: recipes/clinker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (epruesse) [inline recipes/clinker/meta.yaml]: Remove empty line

File: recipes/clinker/clinker-wrapper.sh [hunk 4]
```diff
@@ -15,8 +15,6 @@ END
 )
 # Clinker wrapper - invoke bpipe
 
-#bpipe -p option1="something" -p option2="something_else" [...] $CLINKERDIR/workflow/clinker.pipe /path/to/*.fastq.gz
-# parse and accept parameters in bpipe style, then pass on to fixed clinker.pipe location
```
Comment (epruesse) [inline recipes/clinker/clinker-wrapper.sh]: It appears this entire file should be distributed upstream. Can you create a PR there and link it in the recipe?

File: recipes/clinker/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    from popular fusion finder outputs (JAFFA, tophatFusion, SOAP, deFUSE,
+    Pizzly, etc), that can be then be either viewed in genome viewers such
+    as IGV or through the included plotting feature developed with GViz.
+  license: MIT
+  
+extra:
+  notes: |
+    Wrapper script provided to indicate clinker is a bpipe pipeline,
+    provide example command from wiki, and also a passthrough option.
+    Tests run from /tmp to avoid Docker permission error.
```
Comment (epruesse) [inline recipes/clinker/meta.yaml]: Would you like to add yourself as `recipe-maintainers: [username]`?

File: recipes/clinker/clinker-wrapper.sh (hunk 4 above)
Comment (lonsbio) [reply recipes/clinker/clinker-wrapper.sh:19]: This recipe has been revisited in #25647 . Though will note here, this is same approach as for the [JAFFA](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/jaffa) recipe with wrapper script seperate from source.

**Comment by lonsbio** [comment]
This can be closed, #25647 is latest update to recipe and fixes other issues

**Pull request closed without merging**